src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Strings.resx

```diff
@@ -296,4 +296,8 @@
     <value>A project with ID '{0}' was not found.</value>
     <comment>{0} is the project's unique ID (ProjectGuid)</comment>
   </data>
+  <data name="UnsupportedFeedType" xml:space="preserve">
+    <value>The feed type '{0}' is unsupported.</value>
+    <comment>{0} is the filter type</comment>
+  </data>
```

Comment:
i'm not positive what filter type means (think it is tab name? but i really don't think localizers will understand this.